democracy/views/hearing.py

```diff
@@ -62,23 +44,40 @@ def get_sections(self, hearing):
         serializer.bind('sections', self)  # this is needed to get context in the serializer
         return serializer.to_representation(queryset)
 
+    def get_main_image(self, hearing):
```

Comment:
The performance (or possible lack thereof) of this method slightly worries me. Looks like it might be doing a whole lot of queries and getting lotsa objects just to maybe return one...

```python

main_image_qs = SectionImage.objects.filter(
  section__hearing=hearing,
  section__type__identifier='introduction',
)

if not self.context['request'].user.is_superuser:
  main_image_qs = main_image_qs.filter(published=True)

main_image = main_image_qs.first()
```
would be only one query.